index.html

```diff
@@ -6,6 +6,175 @@
     <title>Homework</title>
 </head>
 <body>
-    
+    <header>
+        <section>
+            <nav>
+                <div>
+                    <img src="icons/Logo.svg" alt="logo" srcset="">
+                </div>
+            </nav>
+        </section>
```

Comment:
тег `section` тут зайвий, тег `nav` не за призначенням, а `div` треба замінити на `a`
зазвичай, майже завжди, іконка сайта є посиланням на його головнку сторінку
```suggestion
        <a href="/">
            <img src="icons/Logo.svg" alt="logo" srcset="">
        </a>
```